File: Tasks/Java Work/CharacterClassifier.java
import java.util.Scanner;
	public class CharacterClassifier{
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);

System.out.println("Enter a character");
String character = input.next();

char letter = character.charAt(0);
int lengthOfLetter = character.length();

if(Character.isDigit(letter) && lengthOfLetter == 1){
System.out.println("This character is a digit");

} else if (Character.isLetter(letter) && lengthOfLetter == 1){
System.out.println("This character is a letter");

} else if (lengthOfLetter != 1) {
System.out.println("This is not a character");

} else { 
	System.out.println("This character is a special symbol");
}
}
}

File: Tasks/Python Work/character_classifier.py
character = input("Enter a character: ")

try:
	if len(character) == 1:
		digit = int(character)
		print("The character is a digit")
	else: print("This is not a character")

except ValueError:
	if character.isalpha() and len(character) == 1: 
		print("This character is a letter")

	elif len(character) == 1 and not character.isalpha(): print("This character is a special symbol") 